graphql_compiler/schema_generation/utils.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2019-present Kensho Technologies, LLC.
+def amend_custom_scalar_types(schema, scalars):
+    """Amend the serialization, parsing and description of custom scalar types in the schema.
+
+       Information about the fields, (other than the name), of custom GraphQLScalarType objects is
```

Comment:
The text that is both between commas and between parens seems kind of awkward. Maybe rephrase?